src/mumble/os_win.cpp

```diff
@@ -196,8 +196,9 @@ FARPROC WINAPI delayHook(unsigned dliNotify, PDelayLoadInfo pdli) {
 	return 0;
 }
 
+decltype(__pfnDliNotifyHook2) __pfnDliNotifyHook2 = delayHook;
```

Comment:
The commit message links to https://msdn.microsoft.com/en-us/library/z9h1h6ty.aspx?f=255&MSPPError=-2147217396 which has more info.

You can do it both ways...

And yes, the decltype is for the MSVC2015 const version of the pointer.